src/wasm-binary-reader.c

```diff
@@ -1307,55 +1353,63 @@ static void read_function_body(Context* ctx,
         uint32_t value = 0;
         in_i32_leb128(ctx, &value, "i32.const value");
         CALLBACK(on_i32_const_expr, value);
+        CALLBACK_CTX(on_opcode_uint32, value);
         break;
       }
 
       case WASM_OPCODE_I64_CONST: {
         uint64_t value = 0;
         in_i64_leb128(ctx, &value, "i64.const value");
         CALLBACK(on_i64_const_expr, value);
+        CALLBACK_CTX(on_opcode_uint64, value);
         break;
       }
 
       case WASM_OPCODE_F32_CONST: {
         uint32_t value_bits = 0;
         in_f32(ctx, &value_bits, "f32.const value");
         CALLBACK(on_f32_const_expr, value_bits);
+        CALLBACK_CTX(on_opcode_uint32, value_bits);
```

Comment:
I think it would be nicer to special case this, so you can print it as a float too